communitybuildingworkinggroup.md

```diff
@@ -0,0 +1,76 @@
+# Community Building Working Group
+
+The Jupyter Community Building Working Group (JCB) acts on behalf of the Project Jupyter
+Executive Council with the objective of growing, building, and connecting the
+Jupyter community of users and contributors. A targeted outcome of this
+working group's efforts is to improve the diversity, equity, and inclusion of our
+community.
+
+The current members of the Jupyter Community Building Working Group are listed in the [Leadership Directory](people).
+
+## Areas of responsibility
+
+### Jupyter Community Workshops
+
+The Jupyter Community Workshop program was established in 2018 to bring
+together small groups of Jupyter community members and core contributors for
+high-impact strategic work and community engagement on focused topics. These
+in-person events are particularly useful for tackling challenging development
+and design projects, growing the community of contributors, and strengthening
+collaborations.
+
+This working group will partner with NumFOCUS to manage the Jupyter Community
+Workshop program operations and budget.
+
+### JupyterCon
+
+Global JupyterCon conferences provide opportunities for the Jupyter community
+to come together to learn and share.
+
+This working group will drive the strategic direction and broad operational
+aspects of JupyterCon, including the partnership with NumFOCUS and the
+community leadership for the event.
+
+### Jupyter Community Events Staff
+
+This working group will be responsible for the relationship with professional
+staff (for example, at NumFOCUS) planning Jupyter events, such as community
+workshops and JupyterCon.
+
+### Operations and Budget
+
+This working group will collaborate with NumFOCUS to serve as stewards and
+decision-makers to allocate the Project Jupyter funds currently associated
+with Jupyter community building at NumFOCUS, including donations and revenue
+from Jupyter Community Workshops and JupyterCon.
+
+### Diversity, Equity, Inclusion
+
+Project Jupyter is an engaged and respectful community made up of people from
+all over the world. This working group pledges to foster a productive,
+harassment-free environment for everyone who participates in organizing or
+attending our events. We recognize our work is a highly visible and
+potentially impactful way for diversity, equity, and inclusion (DEI) efforts
+to be championed within our global community.
+
+In particular, this working group will:
+
+- Collaborate with the Jupyter Diversity, Equity, and Inclusion (DEI)
+  standing committee to find new ways to make a positive impact in
+  Project Jupyter.
+- Partner with the DEI standing committee to train and mentor community event
+  leadership in DEI efforts for their events.
+- Help the DEI standing committee measure impact of DEI efforts in Jupyter events where feasible.
+
+## Reporting
+
+This working group will report to the Executive Council quarterly on all areas of
```

Comment:
We should have a reporting policy that applies to all Working Groups and Standing Committees in a uniform manner. I would rather keep this detail out of the individual charters (but am fine with the quarterly reporting cadence.)

Reply:
Agreed - for a future governance update.